### ISSUE TITLE
module.network_interface_label.provider.null is not present

### ISSUE THREAD

**Issue by aterreno** [cloudposse/terraform-null-label#76]
Hey guys, I've seen similar issues but, is there any way around this, I am on v0.11.14, I even removed the label modules from the codebase but TF keeps complaining about this:

> Error: Error asking for user input: 1 error occurred:
> 	* module.network_interface_label.data.null_data_source.tags_as_list_of_maps[2]: configuration for module.network_interface_label.provider.null is not present; a provider configuration block is required for all operations

This seems to happen either renaming the modules or taking them off completely. 

Thanks

**Comment by aterreno** [comment]
Gotcha, the only way to get rid of those is to first set them as: 
`enabled = false` then they can be removed from the code... 

I'd like to understand why though...

Thanks (feel free to close the issue, I hope it will be useful for others as documentation!)

**Comment by JonHolman** [comment]
I'm getting this message as well, while trying to update our TF syntax to 0.12.  I'm not trying to remove / rename any resources.  Just trying to run `plan` to make sure I caught all of the syntax updates.  At the same time I also updated a mix of terraform-null-label imports of 0.5.3 and 0.11.0 to 0.16.0.  

I also ran `plan` with TF_LOG=DEBUG and then I see several messages "adding implicit provider configuration provider.null, implied first by module." but not all of them.

I'm confused by this message.  Where should I define the null provider?
